templates/partials/question.html

```diff
@@ -3,8 +3,12 @@
 {% from "components/panel/_macro.njk" import onsPanel %}
 
 {% set form = content.form %}
-
-{% set question_title = question.title %}
+{% set interviewer_note = _("Interviewer Note:") %}
+{% if question.interviewer_note %}
+  {% set question_title = '<mark class="instruction">' + interviewer_note + '</mark>' + question.title %}
```

Comment:
Not sure we should be updating the title like this. It feels all of this should be in the macro. The extra param should be passed through onsQuestion and then all it needs is a simple 'if' around mark

Reply:
Moved to macro, thanks!